### ISSUE TITLE
Build library for js with IR

### ISSUE THREAD

**Issue by EmmanueleVilla** [touchlab/Kermit#76]
When using this library with a js(IR) target I have this error:
```
Could not determine the dependencies of task ':shared:jsPackageJson'.
> Could not resolve all dependencies for configuration ':shared:jsNpm'.
   > Could not resolve co.touchlab:kermit:0.1.8.
     Required by:
         project :shared
      > No matching variant of co.touchlab:kermit:0.1.8 was found. The consumer was configured to find a usage of 'kotlin-runtime' of a component, as well as attribute 'org.jetbrains.kotlin.platform.type' with value 'js', attribute 'org.jetbrains.kotlin.js.compiler' with value 'ir'
```
I suppose you just need to add js(IR) to your targets, but I never developed a multiplatform library so I'm not sure :)

**Comment by Shabinder** [comment]
change js to js(BOTH) will lead to make both targets Legacy as well as IR

**Comment by joreilly** [comment]
Also running in to this here trying out new Compose for Web preview version as it needs to use IR compiler.

**Comment by joreilly** [comment]
created https://github.com/touchlab/Kermit/pull/81

**Comment by Shabinder** [comment]
@mrf7 Please create a **New Release** with IR support after having the PR merged.

**Comment by mrf7** [comment]
@Shabinder @joreilly sorry for the delay, the release has been made and these changes are available in 0.1.9. Thanks for the contribution!
